include/eve/arch/cpu/wide.hpp

```diff
@@ -138,7 +141,20 @@ inline namespace EVE_ABI_NAMESPACE
 #if !defined(EVE_DOXYGEN_INVOKED)
     requires( card_base::size() == 2 + sizeof...(vs) )
 #endif
-                  : storage_base(detail::make(eve::as<wide>{}, v0, v1, vs...))
+                  : storage_base(detail::make ( eve::as<wide>{}
+                                              , static_cast<Type>(v0)
+                                              , static_cast<Type>(v1)
+                                              , static_cast<Type>(vs)...
+                                )             )
+    {}
+
+    //! Constructs a eve::wide from a sequence of SIMD product type values
+    template<simd_value S0, simd_value... Ss>
+    EVE_FORCEINLINE wide( S0 v0, Ss... vs) noexcept
```

Comment:
explicit, since it can be of one parameter.

Reply:
Done